### PR TITLE
Minor improvement of diagnostics

### PR DESCRIPTION
**Pull request by Yuriyzabegaev**
## Proposed changes

Following the suggestion of @IvarStefansson, I moved the plotting functionality to a separate function for the default handlers.

## Types of changes

What types of changes does this PR introduce to PorePy?
_Put an `x` in the boxes that apply._

- [ ] Minor change (e.g., dependency bumps, broken links).
- [ ] Bugfix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Testing (contribution related to testing of existing or new functionality).
- [ ] Documentation (contribution related to adding, improving, or fixing documentation).
- [x] Maintenance (e.g., improve logic and performance, remove obsolete code).
- [ ] Other:

## Checklist

_Put an `x` in the boxes that apply or explain briefly why the box is not relevant._

- [x] The documentation is up-to-date.
- [x] Static typing is included in the update.
- [x] This PR does not duplicate existing functionality.
- [x] The update is covered by the test suite (including tests added in the PR).
- [ ] If new skipped tests have been introduced in this PR, `pytest` was run with the `--run-skipped` flag.

### CODE REVIEW

File: src/porepy/viz/diagnostics_mixin.py [hunk 1]
```diff
@@ -90,8 +89,7 @@ def run_diagnostics(
         grouping: GridGroupingType
         | Literal["dense", "subdomains", "interfaces"]
         | None = None,
-        is_plotting_condition_number: bool = False,
-        is_plotting_max: bool = True,
+        default_handlers: tuple[Literal["cond", "max"], ...] = ("max",),
```
Comment (IvarStefansson) [inline src/porepy/viz/diagnostics_mixin.py:92]: Do you want to add a check that it does not contain any other entries?

**Review by IvarStefansson — APPROVED** [review]

File: src/porepy/viz/diagnostics_mixin.py (hunk 1 above)
Comment (IvarStefansson) [reply src/porepy/viz/diagnostics_mixin.py:92]: Also, please comment on the type choice. Are you sure a tuple is appropriate, even though it's length is not fixed?

Comment (Yuriyzabegaev) [reply src/porepy/viz/diagnostics_mixin.py:92]: I agree, this is unnecessary here. I'll change it to Sequence type.

**Comment by Yuriyzabegaev** [comment]
<p>Fixed</p>

---

 <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/869/discussion/#comment-1517935039'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Pull request merged**